README.md

```diff
@@ -139,22 +139,22 @@ You need to load the visualization package, e.g., `using GLMakie`, in your sessi
 
 This can be handy for constructing a path, for example you can interactively set the approximate position and view parameters and then query them for use by the tools above.
 
-```
+```julia
 state = capture_view(scene)
 ```
 
 `state` is a `ViewState` object.
 
 ### Setting the current view state
 
-```
+```julia
 oldstate = set_view!(camera, path, t)
 ```
 
 This updates the current `camera` settings from `path` at time `t`.
 
 ### Displaying the path
 
-```
+```julia
```

Comment:
With the move to Documenter docs, I think we could also slim the README a lot. Otherwise we have two places we have to keep updated.

Reply:
I can definitely slim the README down (and translate the examples to Documenter syntax)!

Reply:
Example translation should be done, but I still have to slim down the README.